### PR TITLE
Bring in fixes from S20

### PR DESCRIPTION
**Pull request by floogulinc**
Bring in relevant fixes from UMM-CSci-3601-S20/lab-0-intro-to-git-everyone#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,9 +270,10 @@ in a bit.)
 
 ### Opening our project in VS Code
 
-Now we'll want to open VS Code and see how we can use it to edit and run our code.
-Launch VS Code, and then choose `File -> Open…`. Navigate to your clone of the repo
-and choose `Open`.
+Now we'll want to open VS Code (search for "Visual Studio Code" in the application
```
Comment (NicMcPhee) [inline README.md:273]: Do we still want the comment about the application menu? Does it make sense of they're running on their own gear?

**Review by NicMcPhee — CHANGES_REQUESTED** [review]
This all looks great except for the one question I had.

File: README.md (hunk 1 above)
Comment (floogulinc) [reply README.md:273]: Yeah that's pretty specific to Gnome. We should assume students know how to open applications on whatever system they're using.

Comment (NicMcPhee) [reply README.md]: I've gone ahead and changed that and I'll go ahead and merge this in.

**Pull request merged**